lib/client/db/dbcmd/dbcmd.go

```diff
@@ -413,6 +412,33 @@ func (c *CLICommandBuilder) getMongoCommand() *exec.Cmd {
 	return c.exe.Command(mongoBin, args...)
 }
 
+func (c *CLICommandBuilder) getMongoAddress() string {
+	query := make(url.Values)
+
+	// Use 5 seconds for server selection timeout to be more align with the
+	// backend. The environment variable serves as a hidden option to force a
+	// different timeout for debugging purpose or extreme situations.
+	serverSelectionTimeoutMS := "5000"
```

Comment:
Can we move the https://github.com/gravitational/teleport/blob/master/lib/srv/db/mongodb/connect.go#L209 to some common package and reuse it here ? I think that the CLI client selection timeout value and selection timeout in teleport mongo engine should be in sync.